app/models/user.rb

```diff
@@ -29,11 +29,12 @@ module Permissions
   end
 
   def role
-    if gds_editor? then "GDS Editor"
-    elsif departmental_editor? then "Departmental Editor"
-    elsif managing_editor? then "Managing Editor"
-    elsif world_editor? then 'World Editor'
-    elsif world_writer? then 'World Writer'
+    case
+    when gds_editor? then "GDS Editor"
+    when departmental_editor? then "Departmental Editor"
+    when managing_editor? then "Managing Editor"
+    when world_editor? then 'World Editor'
+    when world_writer? then 'World Writer'
```

Comment:
This is a bit weird, the previous commit changed this from a `case` to an `if`, and this commit changes it back.  Make up your mind rubocop!  (I don't think there's anything we should do, as TBH I reckon both are readable).